orange/src/main/java/com/hartwig/hmftools/orange/OrangeConfig.java

```diff
@@ -253,39 +208,18 @@ static void registerConfig(@NotNull ConfigBuilder configBuilder)
     @NotNull
     String lilacQcCsv();
 
-    @Nullable
-    String annotatedVirusTsv();
-
-    @Nullable
-    String chordPredictionTxt();
-
-    @Nullable
-    String cuppaResultCsv();
-
-    @Nullable
-    String cuppaSummaryPlot();
-
-    @Nullable
-    String cuppaFeaturePlot();
-
-    @Nullable
-    String cuppaChartPlot();
-
-    @Nullable
-    String peachGenotypeTsv();
-
-    @Nullable
-    String sigsAllocationTsv();
-
     boolean convertGermlineToSomatic();
 
     boolean limitJsonOutput();
 
     boolean addDisclaimer();
 
+    @Nullable
+    OrangeWgsConfig wgsConfig();
+
     default boolean tumorOnlyMode()
     {
-        return referenceSampleId() == null || referenceSampleId().isEmpty();
+        return wgsConfig() == null || wgsConfig().referenceSampleId() == null || wgsConfig().referenceSampleId().isEmpty();
```

Comment:
empty reference sampleId should not be a condition here, that's not really a valid value anyways (it either has to be present or null.